src/kili/mutations/asset/__init__.py

```diff
@@ -240,16 +251,78 @@ def generate_variables(batch: Dict) -> Dict:
         return [item for batch_list in formated_results for item in batch_list]
 
     @typechecked
-    def delete_many_from_dataset(self, asset_ids: List[str]) -> Asset:
+    def change_asset_external_ids(
+        self,
+        asset_ids: List[str],
```

Comment:
we should also include the possibility to give the older asset external_ids

Reply:
done